client.go

```diff
@@ -437,6 +445,27 @@ func (c *client) Range(ctx context.Context, r *RangeOp) ([]DomainObject, string,
 	return objectArray, token, nil
 }
 
+func (c *client) FetchRange(ctx context.Context, r *RangeOp, onNext func(value DomainObject) error) error {
```

Comment:
If I'm calling `FetchRange` with a context, won't there be a pretty good chance the closure I'm passing in already has access to that context?